### ISSUE TITLE
[BUG] withdrawal TX occurs every 10m despite minimum-balance is back above the minimum-balance (10 AKT)

### ISSUE THREAD

**Issue by arno01** [akash-network/node#1579]
Yet to be confirmed but this is what I've been observing earlier today when setting mine to 24h instead of 24h0m0s.

Setting withdrawalperiod=24h via Helm (either in the values.yaml or on the cli via --set ..)

gets into the Pod as expected => OK

```
 root@node1:~# kubectl -n akash-services exec -ti $(kubectl -n akash-services get pods -l app=akash-provider --output jsonpath='{.items[0].metadata.name}') -- env | grep -E 'AKASH_WITH'
 AKASH_WITHDRAWAL_PERIOD=24h
```

However, what I've noticed is this:

```
 AKASH_WITHDRAWAL_PERIOD=24h     => withdrawal is happening every 10 minutes !
 AKASH_WITHDRAWAL_PERIOD=24h0m0s => withdrawal is happening every 24 hours as expected
```

Got drained quit a few of AKT's since yesterday..

```
 root@node1:~# echo "$(echo '"height","code","timestamp","tx","fee","type"'; kubectl -n akash-services logs $(kubectl -n akash-services get pods -l app=akash-provider --output jsonpath='{.items[0].metadata.name}') | grep 'txhash' | grep -Ewo '[0-9A-F]{64}\b' | uniq | while read TX; do akash query tx $TX -o json; done | jq -r '[ .height, .code, .timestamp, .txhash,  (.tx | (.auth_info.fee.amount[0].amount|tonumber / pow(10; 6)), (.body.messages[] | ."@type")) ] | @csv')" | column -t -s ","
 
 "height"   "code"  "timestamp"             "tx"                                                                "fee"     "type"
 ...
 ...
 ...
 "5788554"  0       "2022-05-08T09:12:26Z"  "84FAD9ED1EB18169338E77735A9B97781DA3F168CB966627F04CB6CDB2DF56F1"  0.003653  "/akash.market.v1beta2.MsgWithdrawLease"
 "5788554"  0       "2022-05-08T09:12:26Z"  "64B649D56B2CB6A5F0FC13B0B6FAE46A10BD232223EFB8DA9797D8CADC4D11F0"  0.003667  "/akash.market.v1beta2.MsgWithdrawLease"
 "5788654"  0       "2022-05-08T09:22:28Z"  "57C9175ACD66714262BD2E8E5BD0F2EA56F1C6E79F96198A67E1F5951950483F"  0.003653  "/akash.market.v1beta2.MsgWithdrawLease"
 "5788654"  0       "2022-05-08T09:22:28Z"  "7B887564D94916B1086F931AE443CE4F3439386AA7523F96338834D679394990"  0.003667  "/akash.market.v1beta2.MsgWithdrawLease"
 "5788754"  0       "2022-05-08T09:32:29Z"  "03357EA8119C85053FEE7F6F2F287BF4FE862174ED001E8CE3F05287C35CA171"  0.003653  "/akash.market.v1beta2.MsgWithdrawLease"
 "5788754"  0       "2022-05-08T09:32:29Z"  "81166BEF9C6406A5EF059B9484D518ED403022AC638EA0BCCF93224CEEB0CD5A"  0.003667  "/akash.market.v1beta2.MsgWithdrawLease"
 ```

so am now running helm-charts with  `--set withdrawalperiod=24h0m0s`

just like suggested here:
```
 $ akash provider run --help |grep withdr
       --withdrawal-period duration                  period at which withdrawals are made from the escrow accounts (default 24h0m0s)
```

am not getting drained every 10 mins after that.

**Comment by sadfroghk** [comment]
Greetings, seems I encountered same issue on docker version 

` balance below target amount, withdrawing now module=provider-service cmp=balance-checke`

https://github.com/orgs/ovrclk/packages/container/akash/22445711?tag=0.16.4-rc2

may I know it is fixed on 0.16.4-rc2? or is my config issue..?

Tried both but seems still try to withdraw every 10mins when my provider is below 10atk
(values.yml using in helm deploy)
`#withdrawalperiod: "24h0m0s"
withdrawalperiod: "60m0s"

root@akash-provider-6bfcc8cb5b-d4zg5:/# env|grep -i with 
AKASH_WITHDRAWAL_PERIOD=60m0s
root@akash-provider-6bfcc8cb5b-d4zg5:/# akash version
v0.16.4-rc2
`

**Comment by tidrolpolelsef** [comment]
@troian Is this resolved now?
